py.game-umgebung

n-b-hahn opened the pull request:
The first version of the board now runs.

karlhaensel committed:
add str representation for classes in equation.py

karlhaensel committed:
fix EquationSet.from_json to create class instances for Equations

n-b-hahn committed:
a test comment, to test pusching

n-b-hahn committed:
Gameboard with radom cards

karlhaensel reviewed (changes requested):
Board looks good, thanks! But I think you can remove the first five lines in main.py. The first one triggers a pygame example running before the real board appears and the other lines with print("equation!") were just a placeholder ;)

n-b-hahn committed:
Update main.py

I deleted the first four lines of code because they were not needed to run the code.

n-b-hahn commented:
Thanks for the feedback, I changed it on the py.game-umgebung branch.

karlhaensel reviewed (approved):
Great, thanks a lot! Merge approved.
